src/runtime/parallel/slave.js

```diff
@@ -26,12 +28,26 @@ export default class Slave {
     this.cwd = cwd
     this.eventBroadcaster = new EventEmitter()
     this.stackTraceFilter = new StackTraceFilter()
+
+    const pathSepRegexp = new RegExp(escapeStringRegexp(path.sep), 'g')
+    const pathToRemove =
+      cwd.replace(pathSepRegexp, path.posix.sep) + path.posix.sep
+    this.pathRegexp = new RegExp(escapeStringRegexp(pathToRemove), 'g')
+
     EVENTS.forEach(name => {
-      this.eventBroadcaster.on(name, data =>
-        this.stdout.write(
+      this.eventBroadcaster.on(name, data => {
+        // Format error instance
+        if (data.result && data.result.exception instanceof Error) {
+          data.result.exception = data.result.exception.stack.replace(
+            this.pathRegexp,
+            ''
+          )
```

Comment:
I don't like replacing the error with just the stack as it loses other fields set on the object.